src/GrpcServer/CartaGrpcService.cc

```diff
@@ -21,7 +21,9 @@
 
 uint32_t CartaGrpcService::_scripting_request_id = 0;
 
-CartaGrpcService::CartaGrpcService(std::string auth_token) : _auth_token(auth_token) {}
+CartaGrpcService::CartaGrpcService() : _auth_token(""), _debug_no_auth(true) {}
+
+CartaGrpcService::CartaGrpcService(std::string auth_token) : _auth_token(auth_token), _debug_no_auth(false) {}
```

Comment:
a neater way to do this might be to just pass in an empty auth token when `debug_no_auth` is `true`. That would be consistent with `SimpleFrontendServer` as well

Reply:
I thought that an explicit solution would be better, but I like consistency -- I will change it.